lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -211,6 +210,7 @@ class LighthouseReportViewer {
     logger.log(warnMsg, false);
 
     // Place report in IDB, then navigate current tab to the legacy viewer
+    // TODO: viewer4x :)
```

Comment:
i know this one is probably my fault but can you expand the comment?

is it.. "TODO: Handle 4x reports if we break viewer compat moving to v5"  ??

Reply:
done